wrappers/javascript/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -405,6 +430,7 @@ export class NodeJSAnoncreds implements Anoncreds {
       revocationRegistryDefinitions,
       revocationRegistryDefinitionIds,
       revocationStatusLists,
+      nonRevokedIntervalOverrideList as unknown as Buffer,
```

Comment:
do you think we can improve all the casts etc.. a bit?

Reply:
Yeah it annoys me a lot... It comes down to ref-napi and struct-napi not being type-compatible (everything does work though...) and struct-napi having very complex types.

We do have to cast, but we might be able to extract some of that to a function and reuse it. 

So, we would have like `createNativeNonRevokedIntervalOverrideList(...) -> Buffer` which does the casting only there.